can-route-pushstate.js

```diff
@@ -168,7 +168,24 @@ if (!isFileProtocol && hasPushstate) {
 			// Fix for IE showing blank host, but blank host means current host.
 			var linksHost = node.host || window.location.host;
 
-			if(node.href === "javascript://") {
+			// href has some JS in it, let it run
+			if(node.href.trim().indexOf('javascript') === 0) {
+				try {
+					eval(node.href);
```

Comment:
if you don't call `ev.preventDefault()` ... which should happen just by returning, won't it do the browser's default behavior?

Also, you might want to check for `javascript:` ... less likely someone has a link like `javascript.com` or something.